tools/simple_hydrog/share/make_simple_hydrog.csh

```diff
@@ -31,19 +31,11 @@
 # =========================================================================
 
 if (`gfdl_platform` == "hpcs-csc") then
-    wipetmp
-else
-    if ($?TMPDIR) then
-        echo WARNING
-        echo "WARNING: Intended for PP/AN use; may not work properly on this site"
-        echo WARNING
-        echo
-    else
-        echo 'FATAL: $TMPDIR needed'
-        exit 1
-    endif
+    echo "WARNING: Intended for PP/AN use; may not work properly on this site"
```

Comment:
I think we want the warning to print if the string does not match. But we also want to check for the `gfdl_platform` not being in the path. So think this will work:

```
if ( ! { gfdl_platform} || `gfdl_platform` != "hpcs-csc") then
    echo "WARNING: Intended for PP/AN use; may not work properly on this site"
endif

Reply:
OK. I'll add the ```! { gfdl_platform} || ``` though I do not think it makes a difference.